src/kagglehub/kaggle_cache_resolver.py

```diff
@@ -65,8 +65,6 @@ def __call__(self, h: ModelHandle, path: Optional[str] = None, *, force_download
         if not os.path.exists(cached_path):
             # Only print this if the model is not already mounted.
             logger.info(f"Mounting files to {cached_path}...", extra={**EXTRA_CONSOLE_BLOCK})
```

Comment:
@neshdev, When you worked on reducing log spam, did you mean to add the `EXTRA_CONSOLE_BLOCK` to line #69 instead?

Reply:
Yes - It was only supposed to log if the cache is not found. I must have flipped the branch.

Reply:
@mattdangerw Can you remove the `extra={**EXTRA_CONSOLE_BLOCK}` at line 67 and add it at line 69 instead. No need to delete line 69.

This means the log will be logged to a file but not shown to the user.

Reply:
@mattdangerw, whenever you get a chance to implement the change ^^, we can include it in our next release: #143

Reply:
done! I think. take a look.